CMakeLists.txt

```diff
@@ -116,7 +121,11 @@ endif()
 
 # Having an object target allows unit tests to reuse already built sources without re-building
 add_library(sdbus-c++-objlib OBJECT ${SDBUSCPP_SRCS})
-target_compile_definitions(sdbus-c++-objlib PRIVATE BUILD_LIB=1 LIBSYSTEMD_VERSION=${LIBSYSTEMD_VERSION})
+target_compile_definitions(sdbus-c++-objlib PRIVATE
+    BUILD_LIB=1
+    LIBSYSTEMD_VERSION=${LIBSYSTEMD_VERSION}
+    SDBUS_${LIBSYSTEMD}
+    SDBUS_HEADER=<${LIBSYSTEMD}/sd-bus.h>)
```

Comment:
I propose making sd-bus include folder variable, so instead of `SDBUS_HEADER` we'd have `SDBUS_INCLUDE_DIR`:

`SDBUS_INCLUDE_DIR=${LIBSYSTEMD}`

and then the usage in remaining files would be:

`#include <SDBUS_INCLUDE_PATH/sd-bus.h>`

Reply:
- basu doesn't provide any other header except `sd-bus.h`
- `<SDBUS_INCLUDE_PATH/sd-bus.h>` isn't accepted by C preprocessor

```c
$ echo '#include <DIR/types.h>' >a.c
$ cc -DDIR=sys -c a.c
a.c:1:10: fatal error: 'DIR/types.h' file not found
#include <DIR/types.h>
         ^~~~~~~~~~~~~
1 error generated.
```

Reply:
> basu doesn't provide any other header except `sd-bus.h`

Nevermind, I misremembered:
```
$ pkg info -l basu
basu-0.2.1:
        /usr/local/bin/basuctl
        /usr/local/include/basu/_sd-common.h
        /usr/local/include/basu/sd-bus-protocol.h
        /usr/local/include/basu/sd-bus-vtable.h
        /usr/local/include/basu/sd-bus.h
        /usr/local/include/basu/sd-id128.h
        /usr/local/lib/libbasu.so
        /usr/local/lib/libbasu.so.0
        /usr/local/libdata/pkgconfig/basu.pc
        /usr/local/share/licenses/basu-0.2.1/LGPL21+
        /usr/local/share/licenses/basu-0.2.1/LICENSE
        /usr/local/share/licenses/basu-0.2.1/catalog.mk

```

Reply:
OK, let's keep it as it is.